README.md

```diff
@@ -51,7 +51,13 @@ Import it into your pubspec `dev_dependencies:` section.
 
 A Flutter example app showing how to set up all dependencies and how to use the WidgetDriver framework and how to test your widgets and drivers.
 
-## Start developing
+## Contribute
+
+Once you think you're done coding, open a Pull request onto `master`. PRs can only be merged with a green pipeline. 
+To start the pipeline, you need to apply the `ready for pipeline` label to your PR. This will signify to the pipeline to run the desired checks. 
```

Comment:
Is this true? that as soon as you add the label in an existing PR it will trigger the workflow to try and run again?

Reply:
If what I added to the workflow does what I intend it to, yes. I will have to do some testing in here since I can't test that on my machine

Reply:
Okay cool. I just asked since I don't think it re-triggered my PR when I added the paver afterwards